chapters/bash-advanced.Rmd

```diff
@@ -277,8 +277,10 @@ $ cat summarize_all_books.sh
 ```
 
 ```text
-297 for x in ../data/*.txt; do echo $x; bash book_summary.sh $x Author; done > authors.txt
-298 for x in ../data/*.txt; do echo $x; bash book_summary.sh $x Release; done > releases.txt
+297 for x in ../data/*.txt; do echo $x; \
```

Comment:
```suggestion
297 for x in ../data/*.txt; do echo $x;
```
I'd remove the `\` because if you cut and pasted that command it wouldn't run because the shell doesn't recognise `\` as a line break.